test/specs/ValueSelectionSpec.js

```diff
@@ -0,0 +1,42 @@
+/*
+  *************************
+
+  Value Selection Tests
+
+  *************************
+
+  This spec has tests for checking if a specific value can be selected, given certain slider options.
+  
+*/
+describe("Value Selection Tests", function() {
+
+  var testSlider;
+
+  it("Should slide to the 4200 when slider handle is dragged to the right most.", function() {
```

Comment:
so this test is not actually doing what this description says.

Currently, the code in this test is doing the following:
- Initializing a slider that starts at the value `4200`
- Creating a mouse event with no positional data specified
- triggering the mouse event
- The test is passing because you initialized the slider at the maximum value. so `expect(slideEvt.value).toBe(4200)` will always pass.


What you want it do is this:
- Initialize the slider at some starting `value` that is not the maximum
- Create a mouse event with positional information specified that would slide the handle to the rightmost point
- then call the `getValue` method of the slider, and assert to see if the slider is at the maximum possible value